src/components/ImageInput/style.css

```diff
@@ -24,29 +28,53 @@
 	display: none;
 }
 
-.image-container {
-	position: relative;
+.dropzone .images {
+	padding: 0 !important;
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
 }
 
-.image-container img {
+.images .image-container {
+	display: flex;
+	justify-content: center;
+
+	position: relative;
+	height: fit-content;
+	padding: 0 !important;
 	width: 100%;
 }
 
+.images .image-container .preview-image {
+	width: 100% !important;
+}
+
 .image-container .remove-image-button {
-	width: fit-content;
 	position: absolute;
-
+	
 	display: flex;
 	justify-content: center;
 	align-items: center;
-
-	bottom: 10px;
-	right: 10px;
-
-	background-color: rgb(221, 13, 13);
+	
+	top: 0px;
+	left: 0px;
+	width: 100%;
+	height: 100%;
+	padding: 0 !important;
+	
+	background-color: transparent;
+	opacity: 0;
 	cursor: pointer;
+	transition: 300ms;
 }
 
 .image-container .remove-image-button:hover {
-	background-color: rgb(255, 0, 0);
+	background-color: #009aa857;
+	opacity: 1;
+	transition: 300ms;
 }
+
+.image-container .remove-image-button img{
+	max-height: 100% !important;
+	width: auto !important;
```

Comment:
Is it necessary to use `!important`?

Reply:
In this project I have set the general style to apply to some elements, and it may have higher priority in CSS, so in order to apply some styles directly to some elements, I use `!important`

Reply:
Instead of !important, we can make use of the priority order in CSS to apply the styles instead. From what you have explained, there will be another question: What if I have another component or other components that accidentally share the same class with this one but different style, and then you have to use !important again, so what will the output be and which one will be prioritized to be applied?

Reply:
Noted, I will check the priorities of styles instead of using !important recklessly.